drivers/sensor/adt7420/adt7420.h

```diff
@@ -84,8 +85,9 @@ struct adt7420_dev_config {
 	const char *i2c_port;
 	u16_t i2c_addr;
 #ifdef CONFIG_ADT7420_TRIGGER
-	const char *gpio_port;
-	u8_t int_gpio;
+	gpio_pin_t int_pin;
+	gpio_flags_t int_flags;
+	const char *int_label;
```

Comment:
`int_label` as a name is not very meaningful. `drivers/sensor/lsm6dso/lsm6dso_trigger.c` is using `int_gpio_port`, other drivers tend to use `_name`, not `_label`. Let's use one of the existing conventions. Otherwise, if we prefer to use a new one, let's come up with a better naming scheme.

Reply:
There is no consistency, but `_name` seems to be somewhat more common than others when referring to a specific signal, so I've added this sensor to that group.